index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Vicente Flores Portafolio</title>
+    <meta charset="UTF-8" />
+    <meta name="description" content="This is just me and my experience" />
+    <meta name="author" content="Vicente Flores" />
+  </head>
+
+  <body>
+    <h1>Vicente Flores</h1>
+
+    <section id="about">
```

Comment:
I like having lower case for ids, although AI reviewer mentions it should be uppercase.